build.cake

```diff
@@ -16,8 +16,8 @@ var ErrorDetail = new List<string>();
 // SET PACKAGE VERSION
 //////////////////////////////////////////////////////////////////////
 
-var version = "3.3.0";
-var modifier = "";
+var version = "3.4.0";
+var modifier = "-alpha-1";
```

Comment:
No, that was left over from the alpha release of the portable agent which is in the other repo now. I will remove it.